packages/config/src/mk/openapi.mk

```diff
@@ -1,9 +1,14 @@
 OPENAPI_MAKEFILE := $(lastword $(MAKEFILE_LIST))
+OPENAPI_MAKEFILE_DIR := $(dir $(abspath $(OPENAPI_MAKEFILE)))
+OPENAPI_BIN_DIR := $(dir $(abspath $(OPENAPI_MAKEFILE_DIR)))bin
+
+NODE:=node
+OPENAPI_OVERLAY:=$(NODE) $(OPENAPI_BIN_DIR)/openapi-overlay.ts
 
 OPENAPI_TS ?= $(shell $(MAKE) resolve/bin BIN=openapi-typescript)
 OPENAPI_FORMAT ?= $(shell $(MAKE) resolve/bin BIN=openapi-format)
 
-OVERLAY_SRC_FILES := $(shell find ./src -name "*.overlay.yaml" -type f)
+OVERLAY_SRC_FILES := $(shell find ./src -name "*.overlay.yaml" -type f | sort)
```

Comment:
The `sort` here makes sure that the almost-final `openapi.overlay.yaml` is the last thing to be overlaid.